samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj

```diff
@@ -251,9 +252,12 @@
       <DeploymentContent>false</DeploymentContent>
     </Text>
   </ItemGroup>
+  <ItemGroup>
+    <Media Include="..\..\..\..\deps\3rdparty\openal-soft-winphone\examples\DST-10Class.WAV" />
+  </ItemGroup>
```

Comment:
It turns out that this is in fact problematic.  If you create an SDK package, extract it and build the sample, it wont be able to reference this file.  My recommendation is to have a file added to this project as assets instead of referring to this remotely.  This breaks the CI build.